samples/ImplicitFlow01/AppWithFetchClient/Controllers/AuthorizationController.cs

```diff
@@ -0,0 +1,125 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Text;
+using System.Threading.Tasks;
+using AspNet.Security.OpenIdConnect.Extensions;
+using AspNet.Security.OpenIdConnect.Primitives;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using OpenIddict.Abstractions;
+using OpenIddict.Core;
+using OpenIddict.EntityFrameworkCore.Models;
+using OpenIddict.Server;
+
+public class AuthorizationController : Controller
+{
+    private readonly IOptions<IdentityOptions> _identityOptions;
+    private readonly OpenIddictScopeManager<OpenIddictScope> _scopeManager;
+    private readonly SignInManager<IdentityUser> _signInManager;
+    private readonly UserManager<IdentityUser> _userManager;
+
+    public AuthorizationController(
+        IOptions<IdentityOptions> identityOptions,
+        OpenIddictScopeManager<OpenIddictScope> scopeManager,
+        SignInManager<IdentityUser> signInManager,
+        UserManager<IdentityUser> userManager)
+    {
+        _identityOptions = identityOptions;
+        _scopeManager = scopeManager;
+        _signInManager = signInManager;
+        _userManager = userManager;
+    }
+
+    [HttpGet("~/connect/authorize")]
+    public async Task<IActionResult> Authorize(OpenIdConnectRequest oidcRequest)
+    {
+        if (!User.Identity.IsAuthenticated)
+        {
+            // If the client application request promptless authentication,
+            // return an error indicating that the user is not logged in.
+            if (oidcRequest.HasPrompt(OpenIdConnectConstants.Prompts.None))
```

Comment:
Should we update that to disallow "normal authorization requests" as this first-party-oriented sample only uses `prompt=none`?

Reply:
Good idea. It tightens things up. (TODO).

Reply:
Done.